atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,241 @@
+        """
+        for text in texts:
+            text_id, embedding = generate_embeddings(str(uuid.uuid4()), text)
+            properties = {"properties": [{
+                "text_id": text_id,
+                "text": text,
+                "competency_ids": []
+            }]}
+            self.weaviate_client.add_embeddings(CollectionNames.TEXT.value, embedding, properties)
+
+
+    def initial_cluster_to_competencyPipeline(self):
```

Ruff (0.11.9)

Function name `initial_cluster_to_competencyPipeline` should be lowercase

(N802)